vetspire_interview/test/vetspire_interview/dogs_test.exs

```diff
@@ -0,0 +1,41 @@
+defmodule VetspireInterview.DogsTest do
+  use VetspireInterview.DataCase
+
+  alias VetspireInterview.Dogs
+  alias VetspireInterview.Dogs.Dog
+
+  describe "dogs" do
+    test "list_dogs/0 returns all dogs" do
+      dogs = Dogs.list_dogs()
+      assert Enum.count(dogs) == 10
```

Comment:
This fails saying I had 11, but I may have done something weird, all else looks good, I like the amount of test coverage overall. Maybe some brittleness in the assertions, but I think pretty minor.